package.json

```diff
@@ -54,57 +56,49 @@
     "regenerator-runtime": "^0.13.11",
     "sbc-common-components": "3.0.8",
     "uuid": "^9.0.0",
-    "vue": "2.7.10",
+    "vue": "2.7.14",
```

Comment:
I think this was the problematic vue version? that would hang our builds?

https://github.com/bcgov/business-edit-ui/commit/a35a9ce0cb56645a72faaf5b53f12e9b6b69fb85

Reply:
Yes Travis! However, this version works perfectly well with Vite. Builds are super fast.

Reply:
Did you notice it builds some modules as you need them? So, locally, some things build on the fly.